File: DiagonalDifference.cpp
int diagonalDifference(vector<vector<int>> arr) {
    int sum1 = 0;
    int sum2 = 0;
    for(int i = 0; i < arr.size(); i++){
        for(int j = 0; j < arr.size(); j++){
            if(i == j)
                sum1 += arr[i][j];
            if(i+j == arr.size()-1)
                sum2 += arr[i][j];
        }
    }
    int sum = abs(sum1 - sum2);
    return sum;
}

File: Diagonal_Difference.py
def diagonalDifference(arr):
    num1 = 0
    num2 = 0
    n = len(arr)
    for i in range (n):
        num1 += arr[i][i]
        num2 += arr[n - 1 - i][i]
        
    return (abs(num1 - num2))
    
